bolt-cli/src/cli.rs

```diff
@@ -413,6 +426,29 @@ pub struct DirkOpts {
     pub tls_credentials: TlsCredentials,
 }
 
+/// Options for connecting to a Web3Signer keystore.
+#[derive(Debug, Clone, Parser)]
+pub struct Web3SignerOpts {
+    /// The URL of the Web3Signer keystore.
+    #[clap(long, env = "WEB3SIGNER_URL")]
+    pub url: String,
+
+    /// The TLS credentials for connecting to the Web3Signer keystore.
+    #[clap(flatten)]
+    pub tls_credentials: RustTlsCredentials,
+}
+
+/// TLS credentials which are used for rustls-tls when connecting to reqwest.
+#[derive(Debug, Clone, PartialEq, Eq, Parser)]
+pub struct RustTlsCredentials {
```

Comment:
Not ideal duplicating this, but `reqwest` with `rustls` backend behaves differently, especially with identities https://docs.rs/reqwest/latest/reqwest/tls/struct.Identity.html 😢